src/main/java/io/quarkus/search/app/fetching/FetchingConfig.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.search.app.fetching;
+
+import java.time.Duration;
+import java.util.OptionalInt;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "fetching")
+public interface FetchingConfig {
+    OptionalInt parallelism();
+
+    @WithDefault("30s")
```

Comment:
Just double checking, in the test configs you are passing it as `PT30s` and here just `30s` do both formats work?

Reply:
Yes they should both work. Though `30s` is a Quarkus-specific format that doesn't seem to work in `@QuarkusComponentTest`; I still need to report that.